lib/mutant/color.rb

```diff
@@ -24,7 +24,7 @@ def format(text)
         text
       end
 
-    private
+    private # rubocop:disable AccessModifierIndentation
```

Comment:
@mbj For the sake of consistent indentation rules would you prefer either of the following?

```ruby
module Mutant
  class Color

    # ...

    NONE =
      Class.new(self) do

        # Format null color
        #
        # @param [String] text
        #
        # @return [String]
        #   the argument string
        def format(text)
          text
        end

      private

        # Initialize null color
        #
        # @return [undefined]
        def initialize
        end

      end.new

  end
end
```

or

```ruby
module Mutant
  class Color

    # ...

    NONE = Class.new(self) do

             # Format null color
             #
             # @param [String] text
             #
             # @return [String]
             #   the argument string
             def format(text)
               text
             end

           private

             # Initialize null color
             #
             # @return [undefined]
             def initialize
             end

           end.new

  end
end
```

I personally prefer the first case.